### PR TITLE
Add upgrade guide for Gluster 3.8

### PR DESCRIPTION
**Pull request by nixpanic**
BUG: 1379973
Issue: #133
Signed-off-by: Niels de Vos <ndevos@redhat.com>

### CODE REVIEW

File: Upgrade-Guide/upgrade_to_3.8.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Upgrade procedure from Gluster 3.7.x
+
+The steps to uprade to Gluster 3.8 are the same as for upgrading to Gluster
+3.9. Please follow the detailed instructions from [the 3.9 upgrade
+guide](upgrade_to_3.8.md).
```
Comment (prashanthpai) [inline Upgrade-Guide/upgrade_to_3.8.md]: Is this right ? The link points to `upgrade_to_3.8.md` while the text of the link says `the 3.9 upgrade guide`

**Comment by prashanthpai** [comment]
I think you should also add this new file as an entry in [mkdocs.yml](https://github.com/gluster/glusterdocs/blob/master/mkdocs.yml#L99) and link to this new file from the [upgrade page](https://github.com/gluster/glusterdocs/blob/master/Upgrade-Guide/README.md)

**Comment by humblec** [comment]
@nixpanic @prashanthpai Linking to a later release upgrade doc looks weird to me. If it remains same, we could just copy paste and keep it.

File: Upgrade-Guide/upgrade_to_3.8.md (hunk 1 above)
Comment (nixpanic) [inline Upgrade-Guide/upgrade_to_3.8.md]: Oh, yes, that is wrong.

**Comment by nixpanic** [comment]
@humblec I would like to prevent duplication... Should I move the
contents out of the 3.9 version and link 3.9 to 3.8?

**Comment by humblec** [comment]
@nixpanic ideally 3.8 upgrade docs would have been made available at 3.8 release. So, having a full document for 3.8 is kind of a must thing.

**Pull request merged**